src/cards.json

```diff
@@ -114,5 +117,60 @@
   { "title": "飢渴難耐的Aili", "type": "monster", "src":"https://i.imgur.com/8vVsIYxh.jpg"},
   { "title": "戒色第100天", "type": "trap", "src":"https://imgur.dcard.tw/trx1tL8h.png"},
   { "title": "眾基首", "type": "monster", "src":"https://i.imgur.com/VieLwoz.jpg"},
-  { "title": "斯拉夫之基", "type": "monster", "src":"https://i.imgur.com/iIQTj4s.jpg"}
+  { "title": "斯拉夫之基", "type": "monster", "src":"https://i.imgur.com/iIQTj4s.jpg"},
+  { "title": "一起色色", "type": "magic", "src":"https://i.imgur.com/RLJksCn.jpeg"},
+  { "title": "色吉是空", "type": "trap", "src":"https://i.imgur.com/wkun4qv.jpeg"},
+  { "title": "空吉是色", "type": "trap", "src":"https://i.imgur.com/NtJNqH0.jpg"},
+  { "title": "還是不能色色", "type": "monster", "src":"https://i.imgur.com/S9LrRyq.jpeg"},
+  { "title": "防色色辣椒水", "type": "monster", "src":"https://i.imgur.com/K5xljtO.jpg"},
+  { "title": "不要廢話，直接色色", "type": "monster", "src":"https://i.imgur.com/3Js5OJm.gif"},
+  { "title": "妖艷翹臀色狗狗", "type": "monster", "src":"https://i.imgur.com/hKu4c50.jpg"},
+  { "title": "超强力・「抗色色藥」！", "type": "monster", "src":"https://i.imgur.com/wbbRzuV.jpg"},
+  { "title": "媽的再給我色色試試看", "type": "trap", "src":"https://i.imgur.com/hObyxxM.jpg"},
+  { "title": "現在可以色色了", "type": "monster", "src":"https://i.imgur.com/cmcQcq5.jpg"},
+  { "title": "No.86不色色勇士 柴犬之槍", "type": "monster", "src":"https://i.imgur.com/KJWDdtP.jpg"},
+  { "title": "覺醒的柴犬騎士——Mia", "type": "monster", "src":"https://i.imgur.com/xTXX4iN.jpg"},
+  { "title": "地板很滑的", "type": "monster", "src":"https://i.imgur.com/NZQUj5e.jpg"},
+  { "title": "色色加班", "type": "trap", "src":"https://img.youxi369.com/article/contents/2021/11/06/small_20211106101600299.jpg"},
+  { "title": "電眼光波卡", "type": "trap", "src":"https://img.youxi369.com/article/contents/2021/11/06/small_2021110610160050.jpg"},
+  { "title": "反制可以色色", "type": "trap", "src":"https://img.youxi369.com/article/contents/2021/11/06/small_20211106101559858.jpg"},
+  { "title": "可以色色吗", "type": "trap", "src":"https://mmbiz.qpic.cn/sz_mmbiz_gif/8icRhH8vGrtxIlwf0LfZHiao3a1f6ibfdY1RLObia9EWQonGguN5bUSZQA4ooIic1gvZwRlHttl9iawDwq9Xh8JkEy6A/640"},
```

Comment:
These two gif images from mmbiz.qpic.cn can't be referred by the project webpage, but can be accessed by entering the URL into the address bar of the browser.

Reply:
It will be nice to have gif image, but as I said I prefer to use images with full size Yu-Gi-Oh card frame.